SYCL/SpecConstants/2020/non_native/SpecConstBuffer.cpp

```diff
@@ -0,0 +1,25 @@
+// REQUIRES: opencl-aot, cpu
+
+// RUN: %clangxx -fsycl -fsycl-targets=spir64_x86_64 %S/Inputs/common.cpp -o %t.out \
+// RUN:          -fsycl-dead-args-optimization
+// RUN: env SYCL_PI_TRACE=-1 %CPU_RUN_PLACEHOLDER %t.out | FileCheck %s
+
+#include <cmath>
```

Comment:
Is anything from `cmath` being used?

Reply:
No, I'll remove it, because it is unnecessary.